src/components/Popover/index.tsx

```diff
@@ -136,6 +136,11 @@ export interface PopoverProps {
      * Adds an `aria-label` to the dismiss button for accessibility.
      */
     dismissAriaLabel?: string;
+
+    /**
+     * Optional callback called when the popover is opened or closed
+     */
+    onChange?: (visible: boolean) => void;
```

Comment:
Should we rename it to `onClose` and `onOpen` to be more explicit? MaterialUI Popover already has a `onClose` property that we could use to trigger the `onClose`

Reply:
I was thinking about it being more in line with the `ExpandableSection` where there's an `onChange` which is passed true or false, but I guess that only makes sense if the user can control the components' state too. Will see if I can also add a `visible` prop so you can then do something like:

```tsx
const [visible, setVisible] = useState(false);

<Popover visible={visible} onChange={setVisible}> ... </Popover>
```

Reply:
Wow that was a learning experience! Having a `visible: boolean` parameter doesn't work nicely on the first render - ie `visible = true` still doesn't show the popover since the ref is not set until _after_ our first render... Doesn't seem to be a neat way to get that working, so I think best not to expose a `visible` prop :)

Have renamed to `onClose` and `onOpen` :)